### PR TITLE
Fix naming of bearer authorizer

### PR DESCRIPTION
**Pull request by jyunderwood**
Rename OAuth2BearerAuhtorizer to OAuth2BearerAuthorizer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -363,7 +363,7 @@ __authorizers and authenticators have to fit together__.
 
 Ember Simple Auth comes with 2 authorizers:
 
-* [`OAuth2BearerAuthorizer`](http://ember-simple-auth.com/api/classes/OAuth2BearerAuhtorizer.html): an OAuth 2.0 authorizer that uses Bearer tokens
+* [`OAuth2BearerAuthorizer`](http://ember-simple-auth.com/api/classes/OAuth2BearerAuthorizer.html): an OAuth 2.0 authorizer that uses Bearer tokens
```
Comment (marcoow) [inline README.md]: changing this now would result in a broken link until 1.1 has been release with updated docs

Comment (jyunderwood) [inline README.md]: Do you want me to revert the change to the README but leave the change to the class comment?

Comment (marcoow) [inline README.md]: That would be great

**Comment by marcoow** [comment]
👍
